HomepageV2.css

```diff
@@ -0,0 +1,335 @@
+html {
+    scroll-behavior: smooth;
+}
+body {
+    -ms-overflow-style: none;  // IE 10+
+    scrollbar-width: none;  // Firefox
+
+}
+body::-webkit-scrollbar { 
+    display: none;  // Safari and Chrome
+}
+
+@font-face {
+    font-family: "neutra";
+    src: url("NeutraText-book.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "neutra-bold";
+    src: url("NeutraText-bold.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "robot";
+    src: url("Robot Renegades.otf") format("opentype");
+}
+
+body {
+    padding: 0;
+    margin: 0;    
+    font-family: "neutra-bold";
+}
+
+.teamLogo {
+    color: rgb(249, 211, 139);
+    position: relative;
+    top: -31px;
```

Comment:
Using REM is a better practice - allows for less work on transition to smaller screens